src/client/Client.cc

```diff
@@ -6562,15 +6562,15 @@ void Client::tick()
 {
   ldout(cct, 20) << "tick" << dendl;
 
-  utime_t now = ceph_clock_now();
+  ceph::coarse_mono_time now = ceph::coarse_mono_clock::now();
 
   /*
    * If the mount() is not finished
    */
   if (is_mounting() && !mds_requests.empty()) {
     MetaRequest *req = mds_requests.begin()->second;
 
-    if (req->op_stamp + cct->_conf->client_mount_timeout < now) {
+    if (req->coarse_stamp + mount_timeout < now) {
```

Comment:
Where is setting the `req->coarse_stamp` ? 

IMO, it should be in where the `req->op_stamp` is initialized.

Reply:
Added.